BuildingSync.xsd

```diff
@@ -14856,170 +14857,175 @@
-        <xs:enumeration value="Energy generation plant"/>
-        <xs:enumeration value="Industrial manufacturing plant"/>
-        <xs:enumeration value="Utility"/>
-        <xs:enumeration value="Industrial"/>
-        <xs:enumeration value="Agricultural estate"/>
-        <xs:enumeration value="Mixed-use commercial"/>
-        <xs:enumeration value="Parking"/>
-        <xs:enumeration value="Attic"/>
-        <xs:enumeration value="Basement"/>
-        <xs:enumeration value="Dining area"/>
-        <xs:enumeration value="Living area"/>
-        <xs:enumeration value="Sleeping area"/>
-        <xs:enumeration value="Laundry area"/>
-        <xs:enumeration value="Lodging area"/>
-        <xs:enumeration value="Dressing area"/>
-        <xs:enumeration value="Restroom"/>
-        <xs:enumeration value="Auditorium"/>
-        <xs:enumeration value="Classroom"/>
-        <xs:enumeration value="Day room"/>
-        <xs:enumeration value="Sport play area"/>
-        <xs:enumeration value="Stage"/>
-        <xs:enumeration value="Spectator area"/>
-        <xs:enumeration value="Office work area"/>
-        <xs:enumeration value="Non-office work area"/>
-        <xs:enumeration value="Common area"/>
-        <xs:enumeration value="Reception area"/>
-        <xs:enumeration value="Waiting area"/>
-        <xs:enumeration value="Transportation waiting area"/>
-        <xs:enumeration value="Lobby"/>
-        <xs:enumeration value="Conference room"/>
-        <xs:enumeration value="Computer lab"/>
-        <xs:enumeration value="Data center"/>
-        <xs:enumeration value="Printing room"/>
-        <xs:enumeration value="Media center"/>
-        <xs:enumeration value="Telephone data entry"/>
-        <xs:enumeration value="Darkroom"/>
-        <xs:enumeration value="Courtroom"/>
-        <xs:enumeration value="Kitchen"/>
-        <xs:enumeration value="Kitchenette"/>
-        <xs:enumeration value="Refrigerated storage"/>
-        <xs:enumeration value="Bar-Nightclub"/>
-        <xs:enumeration value="Bar"/>
-        <xs:enumeration value="Dance floor"/>
-        <xs:enumeration value="Trading floor"/>
-        <xs:enumeration value="TV studio"/>
-        <xs:enumeration value="Security room"/>
-        <xs:enumeration value="Shipping and receiving"/>
-        <xs:enumeration value="Mechanical room"/>
-        <xs:enumeration value="Chemical storage room"/>
-        <xs:enumeration value="Non-chemical storage room"/>
-        <xs:enumeration value="Janitorial closet"/>
-        <xs:enumeration value="Vault"/>
-        <xs:enumeration value="Corridor"/>
-        <xs:enumeration value="Deck"/>
-        <xs:enumeration value="Courtyard"/>
-        <xs:enumeration value="Atrium"/>
-        <xs:enumeration value="Science park"/>
-        <xs:enumeration value="Other"/>
-        <xs:enumeration value="Unknown"/>
-      </xs:restriction>
-    </xs:simpleType>
   </xs:element>
+  <xs:simpleType name="OccupancyClassifications">
```

Comment:
Please rename according to the naming convention for types: "OccupancyClassificationType".